src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpArgs.java

```diff
@@ -0,0 +1,9 @@
+package ee.buerokratt.ruuter.domain.steps.http;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class HttpArgs {
```

Comment:
Is this kind of a "base object" really required here? If the purpose is to be able to return the "mock" args through HttpStepResult object, then id say that the correct solution would be to convert the "mock args" object to the accepted "query args" object.